<details>

<summary><strong>Theatre: A software tool for detailed comparative analysis and visualization of genomic sequence.</strong></summary>

<br>

**Autores:**<br> Edwards YJ, Carver TJ, Vavouri T, Frith M, Bishop MJ, Elgar G

<br><br>

**Enlace a la revista:** <a href="https://doi.org/10.1093/nar/gkg501" target="_blank">https://doi.org/10.1093/nar/gkg501</a>

<br><br>

**Resumen:**<br> Theatre is a web-based computing system designed for the comparative analysis of genomic sequences, especially with respect to motifs likely to be involved in the regulation of gene expression. Theatre is an interface to commonly used sequence analysis tools and biological sequence databases to determine or predict the positions of coding regions, repetitive sequences and transcription factor binding sites in families of DNA sequences. The information is displayed in a manner that can be easily understood and can reveal patterns that might not otherwise have been noticed. In addition to web-based output, Theatre can produce publication quality colour hardcopies showing predicted features in aligned genomic sequences. A case study using the p53 promoter region of four mammalian species and two fish species is described. Unlike the mammalian sequences the promoter regions in fish have not been previously predicted or characterized and we report the differences in the p53 promoter region of four mammals and that predicted for two fish species. Theatre can be accessed at http://www.hgmp.mrc.ac.uk/Registered/Webapp/theatre/.

<br><br>

</details>

<details>

<summary><strong>Highly conserved non-coding sequences are associated with vertebrate development.</strong></summary>

<br>

**Autores:**<br> Woolfe A, Goodson M, Goode DK, Snell P, McEwen GK, Vavouri T, Smith SF, North P, Callaway H, Kelly K, Walter K, Abnizova I, Gilks W, Edwards YJ, Cooke JE, Elgar G

<br><br>

**Enlace a la revista:** <a href="https://doi.org/SO  - PLoS Biol. 2005 Jan;3(1):e7. doi: 10.1371/journal.pbio.0030007. Epub 2004 Nov 11." target="_blank">https://doi.org/SO - PLoS Biol. 2005 Jan;3(1):e7. doi: 10.1371/journal.pbio.0030007. Epub 2004 Nov 11.</a>

<br><br>

**Resumen:**<br> In addition to protein coding sequence, the human genome contains a significant amount of regulatory DNA, the identification of which is proving somewhat recalcitrant to both in silico and functional methods. An approach that has been used with some success is comparative sequence analysis, whereby equivalent genomic regions from different organisms are compared in order to identify both similarities and differences. In general, similarities in sequence between highly divergent organisms imply functional constraint. We have used a whole-genome comparison between humans and the pufferfish, Fugu rubripes, to identify nearly 1,400 highly conserved non-coding sequences. Given the evolutionary divergence between these species, it is likely that these sequences are found in, and furthermore are essential to, all vertebrates. Most, and possibly all, of these sequences are located in and around genes that act as developmental regulators. Some of these sequences are over 90% identical across more than 500 bases, being more highly conserved than coding sequence between these two species. Despite this, we cannot find any similar sequences in invertebrate genomes. In order to begin to functionally test this set of sequences, we have used a rapid in vivo assay system using zebrafish embryos that allows tissue-specific enhancer activity to be identified. Functional data is presented for highly conserved non-coding sequences associated with four unrelated developmental regulators (SOX21, PAX6, HLXB9, and SHH), in order to demonstrate the suitability of this screen to a wide range of genes and expression patterns. Of 25 sequence elements tested around these four genes, 23 show significant enhancer activity in one or more tissues. We have identified a set of non-coding sequences that are highly conserved throughout vertebrates. They are found in clusters across the human genome, principally around genes that are implicated in the regulation of development, including many transcription factors. These highly conserved non-coding sequences are likely to form part of the genomic circuitry that uniquely defines vertebrate development.

<br><br>

</details>

<details>

<summary><strong>Molecular characterisation of the SAND protein family: a study based on comparative genomics, structural bioinformatics and phylogeny.</strong></summary>

<br>

**Autores:**<br> Cottage A, Mullan L, Portela MB, Hellen E, Carver T, Patel S, Vavouri T, Elgar G, Edwards YJ

<br><br>

**Enlace a la revista:** <a href="No disponible" target="_blank">No disponible</a>

<br><br>

**Resumen:**<br> The activities of vertebrate lysosomes are critical to many essential cellular processes. The yeast vacuole is analogous to the mammalian lysosome and is used as a tool to gain insights into vesicle mediated vacuolar/lysosome transport. The protein SAND, which does not contain a SAND domain (PFAM accession number PF01342), has recently been shown to function at the tethering/docking stage of vacuole fusion as a critical component of the vacuole SNARE complex. In this publication we have identified SAND in diverse eukaryotes, from single celled organisms such as the yeasts to complex multi-cellular chordates such as mammals. We have demonstrated subfamily divisions in the SAND proteins and show that in vertebrates, a duplication event gave rise to two SAND sequences. This duplication appears to have occurred during early vertebrate evolution and conceivably with the evolution of lysosomes. Using bioinformatics we predict a secondary structure, solvent accessibility profile and protein fold for the SAND proteins and determine conserved sequence motifs, present in all SAND proteins and those that are specific to subsets. A comprehensive evaluation of yeast and human functional studies in conjunction with our in silico analysis has identified potential roles for some of these motifs.

<br><br>

</details>

<details>

<summary><strong>Prediction of cis-regulatory elements using binding site matrices--the successes, the failures and the reasons for both.</strong></summary>

<br>

**Autores:**<br> Vavouri T, Elgar G

<br><br>

**Enlace a la revista:** <a href="https://doi.org/10.1016/j.gde.2005.05.002" target="_blank">https://doi.org/10.1016/j.gde.2005.05.002</a>

<br><br>

**Resumen:**<br> Protein-DNA interactions control many aspects of animal development and cellular responses to the environment. Although profiling of individual transcription factor binding sites is not a reliable guide for predicting the position of cis-regulatory elements in large genomes, modelling the evolution and the organization of regulatory elements has provided enough information to make some successful predictions. For vertebrate genomes, the field is limited by the lack of sufficient experimental data upon which to build reliable models. Nonetheless, a combination of experimental, computational and comparative data is likely to reveal aspects of complex regulatory networks in vertebrates, just as it has already done for simple eukaryotic genomes.

<br><br>

</details>

<details>

<summary><strong>Defining a genomic radius for long-range enhancer action: duplicated conserved non-coding elements hold the key.</strong></summary>

<br>

**Autores:**<br> Vavouri T, McEwen GK, Woolfe A, Gilks WR, Elgar G

<br><br>

**Enlace a la revista:** <a href="https://doi.org/SO  - Trends Genet. 2006 Jan;22(1):5-10. doi: 10.1016/j.tig.2005.10.005. Epub 2005 Nov  10." target="_blank">https://doi.org/SO - Trends Genet. 2006 Jan;22(1):5-10. doi: 10.1016/j.tig.2005.10.005. Epub 2005 Nov 10.</a>

<br><br>

**Resumen:**<br> Many conserved non-coding elements (CNEs) in vertebrate genomes have been shown to function as tissue-specific enhancers. However, the target genes of most CNEs are unknown. Here we show that the target genes of duplicated CNEs can be predicted by considering their neighbouring paralogous genes. This enables us to provide the first systematic estimate of the genomic range for distal cis-regulatory interactions in the human genome: half of CNEs are \>250 kb away from their associated gene.

<br><br>

</details>

<details>

<summary><strong>Ancient duplicated conserved noncoding elements in vertebrates: a genomic and functional analysis.</strong></summary>

<br>

**Autores:**<br> McEwen GK, Woolfe A, Goode D, Vavouri T, Callaway H, Elgar G

<br><br>

**Enlace a la revista:** <a href="https://doi.org/SO  - Genome Res. 2006 Apr;16(4):451-65. doi: 10.1101/gr.4143406. Epub 2006 Mar 13." target="_blank">https://doi.org/SO - Genome Res. 2006 Apr;16(4):451-65. doi: 10.1101/gr.4143406. Epub 2006 Mar 13.</a>

<br><br>

**Resumen:**<br> Fish-mammal genomic comparisons have proved powerful in identifying conserved noncoding elements likely to be cis-regulatory in nature, and the majority of those tested in vivo have been shown to act as tissue-specific enhancers associated with genes involved in transcriptional regulation of development. Although most of these elements share little sequence identity to each other, a small number are remarkably similar and appear to be the product of duplication events. Here, we searched for duplicated conserved noncoding elements in the human genome, using comparisons with Fugu to select putative cis-regulatory sequences. We identified 124 families of duplicated elements, each containing between two and five members, that are highly conserved within and between vertebrate genomes. In 74% of cases, we were able to assign a specific set of paralogous genes with annotation relating to transcriptional regulation and/or development to each family, thus removing much of the ambiguity in identifying associated genes. We find that duplicate elements have the potential to up-regulate reporter gene expression in a tissue-specific manner and that expression domains often overlap, but are not necessarily identical, between family members. Over two thirds of the families are conserved in duplicate in fish and appear to predate the large-scale duplication events thought to have occurred at the origin of vertebrates. We propose a model whereby gene duplication and the evolution of cis-regulatory elements can be considered in the context of increased morphological diversity and the emergence of the modern vertebrate body plan.

<br><br>

</details>

<details>

<summary><strong>Parallel evolution of conserved non-coding elements that target a common set of developmental regulatory genes from worms to humans.</strong></summary>

<br>

**Autores:**<br> Vavouri T, Walter K, Gilks WR, Lehner B, Elgar G

<br><br>

**Enlace a la revista:** <a href="https://doi.org/10.1186/gb-2007-8-2-r15" target="_blank">https://doi.org/10.1186/gb-2007-8-2-r15</a>

<br><br>

**Resumen:**<br> BACKGROUND: The human genome contains thousands of non-coding sequences that are often more conserved between vertebrate species than protein-coding exons. These highly conserved non-coding elements (CNEs) are associated with genes that coordinate development, and have been proposed to act as transcriptional enhancers. Despite their extreme sequence conservation in vertebrates, sequences homologous to CNEs have not been identified in invertebrates. RESULTS: Here we report that nematode genomes contain an alternative set of CNEs that share sequence characteristics, but not identity, with their vertebrate counterparts. CNEs thus represent a very unusual class of sequences that are extremely conserved within specific animal lineages yet are highly divergent between lineages. Nematode CNEs are also associated with developmental regulatory genes, and include well-characterized enhancers and transcription factor binding sites, supporting the proposed function of CNEs as cis-regulatory elements. Most remarkably, 40 of 156 human CNE-associated genes with invertebrate orthologs are also associated with CNEs in both worms and flies. CONCLUSION: A core set of genes that regulate development is associated with CNEs across three animal groups (worms, flies and vertebrates). We propose that these CNEs reflect the parallel evolution of alternative enhancers for a common set of developmental regulatory genes in different animal groups. This 're-wiring' of gene regulatory networks containing key developmental coordinators was probably a driving force during the evolution of animal body plans. CNEs may, therefore, represent the genomic traces of these 'hard-wired' core gene regulatory networks that specify the development of each alternative animal body plan.

<br><br>

</details>

<details>

<summary><strong>A simple principle concerning the robustness of protein complex activity to changes in gene expression.</strong></summary>

<br>

**Autores:**<br> Semple JI, Vavouri T, Lehner B

<br><br>

**Enlace a la revista:** <a href="https://doi.org/CIN - BMC Syst Biol. 2009 Jan 30;3:16. doi: 10.1186/1752-0509-3-16. PMID: 19183469" target="_blank">https://doi.org/CIN - BMC Syst Biol. 2009 Jan 30;3:16. doi: 10.1186/1752-0509-3-16. PMID: 19183469</a>

<br><br>

**Resumen:**<br> BACKGROUND: The functions of a eukaryotic cell are largely performed by multi-subunit protein complexes that act as molecular machines or information processing modules in cellular networks. An important problem in systems biology is to understand how, in general, these molecular machines respond to perturbations. RESULTS: In yeast, genes that inhibit growth when their expression is reduced are strongly enriched amongst the subunits of multi-subunit protein complexes. This applies to both the core and peripheral subunits of protein complexes, and the subunits of each complex normally have the same loss-of-function phenotypes. In contrast, genes that inhibit growth when their expression is increased are not enriched amongst the core or peripheral subunits of protein complexes, and the behaviour of one subunit of a complex is not predictive for the other subunits with respect to over-expression phenotypes. CONCLUSION: We propose the principle that the overall activity of a protein complex is in general robust to an increase, but not to a decrease in the expression of its subunits. This means that whereas phenotypes resulting from a decrease in gene expression can be predicted because they cluster on networks of protein complexes, over-expression phenotypes cannot be predicted in this way. We discuss the implications of these findings for understanding how cells are regulated, how they evolve, and how genetic perturbations connect to disease in humans.

<br><br>

</details>

<details>

<summary><strong>Tuning in to the signals: noncoding sequence conservation in vertebrate genomes.</strong></summary>

<br>

**Autores:**<br> Elgar G, Vavouri T

<br><br>

**Enlace a la revista:** <a href="https://doi.org/SO  - Trends Genet. 2008 Jul;24(7):344-52. doi: 10.1016/j.tig.2008.04.005. Epub 2008  May 29." target="_blank">https://doi.org/SO - Trends Genet. 2008 Jul;24(7):344-52. doi: 10.1016/j.tig.2008.04.005. Epub 2008 May 29.</a>

<br><br>

**Resumen:**<br> Aligning and comparing genomic sequences enables the identification of conserved sequence signatures and can enrich for coding and noncoding functional regions. In vertebrates, the comparison of human and rodent genomes and the comparison of evolutionarily distant genomes, such as human and pufferfish, have identified specific sets of 'ultraconserved' sequence elements associated with the control of early development. However, is this just the tip of a 'conservation iceberg' or do these sequences represent a specific class of regulatory element? Studies on the zebrafish phox2b gene region and the ENCODE project suggest that many regulatory elements are not highly conserved, posing intriguing questions about the relationship between noncoding sequence conservation and function and the evolution of regulatory sequences.

<br><br>

</details>

<details>

<summary><strong>Widespread conservation of genetic redundancy during a billion years of eukaryotic evolution.</strong></summary>

<br>

**Autores:**<br> Vavouri T, Semple JI, Lehner B

<br><br>

**Enlace a la revista:** <a href="https://doi.org/SO  - Trends Genet. 2008 Oct;24(10):485-8. doi: 10.1016/j.tig.2008.08.005. Epub 2008  Sep 9." target="_blank">https://doi.org/SO - Trends Genet. 2008 Oct;24(10):485-8. doi: 10.1016/j.tig.2008.08.005. Epub 2008 Sep 9.</a>

<br><br>

**Resumen:**<br> Genetic redundancy means that two genes can perform the same function. Using a comprehensive phylogenetic analysis, we show here in both Saccharomyces cerevisiae and Caenorhabditis elegans that genetic redundancy is not just a transient consequence of gene duplication, but is often an evolutionary stable state. In multiple examples, genes have retained redundant functions since the divergence of the animal, plant and fungi kingdoms over a billion years ago. The stable conservation of genetic redundancy contrasts with the more rapid evolution of genetic interactions between unrelated genes and can be explained by theoretical models including a 'piggyback' mechanism in which overlapping redundant functions are co-selected with nonredundant ones.

<br><br>

</details>

<details>

<summary><strong>Conserved noncoding elements and the evolution of animal body plans.</strong></summary>

<br>

**Autores:**<br> Vavouri T, Lehner B

<br><br>

**Enlace a la revista:** <a href="https://doi.org/10.1002/bies.200900014" target="_blank">https://doi.org/10.1002/bies.200900014</a>

<br><br>

**Resumen:**<br> The genomes of vertebrates, flies, and nematodes contain highly conserved noncoding elements (CNEs). CNEs cluster around genes that regulate development, and where tested, they can act as transcriptional enhancers. Within an animal group CNEs are the most conserved sequences but between groups they are normally diverged beyond recognition. Alternative CNEs are, however, associated with an overlapping set of genes that control development in all animals. Here, we discuss the evidence that CNEs are part of the core gene regulatory networks (GRNs) that specify alternative animal body plans. The major animal groups arose \>550 million years ago. We propose that the cis-regulatory inputs identified by CNEs arose during the "re-wiring" of regulatory interactions that occurred during early animal evolution. Consequently, different animal groups, with different core GRNs, contain alternative sets of CNEs. Due to the subsequent stability of animal body plans, these core regulatory sequences have been evolving in parallel under strong purifying selection in different animal groups.

<br><br>

</details>

<details>

<summary><strong>Intrinsic protein disorder and interaction promiscuity are widely associated with dosage sensitivity.</strong></summary>

<br>

**Autores:**<br> Vavouri T, Semple JI, Garcia-Verdugo R, Lehner B

<br><br>

**Enlace a la revista:** <a href="https://doi.org/CIN - Cell. 2009 Jul 10;138(1):16-8. doi: 10.1016/j.cell.2009.06.024. PMID: 19596229" target="_blank">https://doi.org/CIN - Cell. 2009 Jul 10;138(1):16-8. doi: 10.1016/j.cell.2009.06.024. PMID: 19596229</a>

<br><br>

**Resumen:**<br> Why are genes harmful when they are overexpressed? By testing possible causes of overexpression phenotypes in yeast, we identify intrinsic protein disorder as an important determinant of dosage sensitivity. Disordered regions are prone to make promiscuous molecular interactions when their concentration is increased, and we demonstrate that this is the likely cause of pathology when genes are overexpressed. We validate our findings in two animals, Drosophila melanogaster and Caenorhabditis elegans. In mice and humans the same properties are strongly associated with dosage-sensitive oncogenes, such that mass-action-driven molecular interactions may be a frequent cause of cancer. Dosage-sensitive genes are tightly regulated at the transcriptional, RNA, and protein levels, which may serve to prevent harmful increases in protein concentration under physiological conditions. Mass-action-driven interaction promiscuity is a single theoretical framework that can be used to understand, predict, and possibly treat the effects of increased gene expression in evolution and disease.

<br><br>

</details>

<details>

<summary><strong>Characterisation of conserved non-coding sequences in vertebrate genomes using bioinformatics, statistics and functional studies.</strong></summary>

<br>

**Autores:**<br> Edwards YJ, Walter K, McEwen G, Vavouri T, Kelly KA, Abnizova I, Woolfe A, Goode DK, Goodson M, North P, Snell P, Callaway H, Smith SF, Gilks WR, Cooke JE, Elgar G

<br><br>

**Enlace a la revista:** <a href="No disponible" target="_blank">No disponible</a>

<br><br>

**Resumen:**<br> We recently identified approximately 1400 conserved non-coding elements (CNEs) shared by the genomes of fugu (Takifugu rubripes) and human that appear to be associated with developmental regulation in vertebrates \[Woolfe, A., Goodson, M., Goode, D.K., Snell, P., McEwen, G.K., Vavouri, T., Smith, S.F., North, P., Callaway, H., Kelly, K., Walter, K., Abnizova, I., Gilks, W., Edwards, Y.J.K., Cooke, J.E., Elgar, G., 2005. Highly conserved non-coding sequences are associated with vertebrate development. PLoS Biol. 3 (1), e7\]. This study encompassed a multi-disciplinary approach using bioinformatics, statistical methods and functional assays to identify and characterise the CNEs. Using an in vivo enhancer assay, over 90% of tested CNEs up-regulate tissue-specific GFP expression. Here we review our group's research in the field of characterising non-coding sequences conserved in vertebrates. We take this opportunity to discuss our research in progress and present some results of new and additional analyses. These include a phylogenomics analysis of CNEs, sequence conservation patterns in vertebrate CNEs and the distribution of human SNPs in the CNEs. We highlight the usefulness of the CNE dataset to help correlate genetic variation in health and disease. We also discuss the functional analysis using the enhancer assay and the enrichment of predicted transcription factor binding sites for two CNEs. Public access to the CNEs plus annotation is now possible and is described. The content of this review was presented by Dr. Y.J.K. Edwards at the TODAI International Symposium on Functional Genomics of the Pufferfish, Tokyo, Japan, 3-6 November 2004.

<br><br>

</details>

<details>

<summary><strong>Predicting genetic modifier loci using functional gene networks.</strong></summary>

<br>

**Autores:**<br> Lee I, Lehner B, Vavouri T, Shin J, Fraser AG, Marcotte EM

<br><br>

**Enlace a la revista:** <a href="https://doi.org/SO  - Genome Res. 2010 Aug;20(8):1143-53. doi: 10.1101/gr.102749.109. Epub 2010 Jun 9." target="_blank">https://doi.org/SO - Genome Res. 2010 Aug;20(8):1143-53. doi: 10.1101/gr.102749.109. Epub 2010 Jun 9.</a>

<br><br>

**Resumen:**<br> Most phenotypes are genetically complex, with contributions from mutations in many different genes. Mutations in more than one gene can combine synergistically to cause phenotypic change, and systematic studies in model organisms show that these genetic interactions are pervasive. However, in human association studies such nonadditive genetic interactions are very difficult to identify because of a lack of statistical power--simply put, the number of potential interactions is too vast. One approach to resolve this is to predict candidate modifier interactions between loci, and then to specifically test these for associations with the phenotype. Here, we describe a general method for predicting genetic interactions based on the use of integrated functional gene networks. We show that in both Saccharomyces cerevisiae and Caenorhabditis elegans a single high-coverage, high-quality functional network can successfully predict genetic modifiers for the majority of genes. For C. elegans we also describe the construction of a new, improved, and expanded functional network, WormNet 2. Using this network we demonstrate how it is possible to rapidly expand the number of modifier loci known for a gene, predicting and validating new genetic interactions for each of three signal transduction genes. We propose that this approach, termed network-guided modifier screening, provides a general strategy for predicting genetic interactions. This work thus suggests that a high-quality integrated human gene network will provide a powerful resource for modifier locus discovery in many different diseases.

<br><br>

</details>

<details>

<summary><strong>Chromatin organization in sperm may be the major functional consequence of base composition variation in the human genome.</strong></summary>

<br>

**Autores:**<br> Vavouri T, Lehner B

<br><br>

**Enlace a la revista:** <a href="https://doi.org/SO  - PLoS Genet. 2011 Apr;7(4):e1002036. doi: 10.1371/journal.pgen.1002036. Epub 2011  Apr 7." target="_blank">https://doi.org/SO - PLoS Genet. 2011 Apr;7(4):e1002036. doi: 10.1371/journal.pgen.1002036. Epub 2011 Apr 7.</a>

<br><br>

**Resumen:**<br> Chromatin in sperm is different from that in other cells, with most of the genome packaged by protamines not nucleosomes. Nucleosomes are, however, retained at some genomic sites, where they have the potential to transmit paternal epigenetic information. It is not understood how this retention is specified. Here we show that base composition is the major determinant of nucleosome retention in human sperm, predicting retention very well in both genic and non-genic regions of the genome. The retention of nucleosomes at GC-rich sequences with high intrinsic nucleosome affinity accounts for the previously reported retention at transcription start sites and at genes that regulate development. It also means that nucleosomes are retained at the start sites of most housekeeping genes. We also report a striking link between the retention of nucleosomes in sperm and the establishment of DNA methylation-free regions in the early embryo. Taken together, this suggests that paternal nucleosome transmission may facilitate robust gene regulation in the early embryo. We propose that chromatin organization in the male germline, rather than in somatic cells, is the major functional consequence of fine-scale base composition variation in the human genome. The selective pressure driving base composition evolution in mammals could, therefore, be the need to transmit paternal epigenetic information to the zygote.

<br><br>

</details>

<details>

<summary><strong>Human genes with CpG island promoters have a distinct transcription-associated chromatin organization.</strong></summary>

<br>

**Autores:**<br> Vavouri T, Lehner B

<br><br>

**Enlace a la revista:** <a href="https://doi.org/10.1186/gb-2012-13-11-r110" target="_blank">https://doi.org/10.1186/gb-2012-13-11-r110</a>

<br><br>

**Resumen:**<br> BACKGROUND: More than 50% of human genes initiate transcription from CpG dinucleotide-rich regions referred to as CpG islands. These genes show differences in their patterns of transcription initiation, and have been reported to have higher levels of some activation-associated chromatin modifications. RESULTS: Here we report that genes with CpG island promoters have a characteristic transcription-associated chromatin organization. This signature includes high levels of the transcription elongation-associated histone modifications H4K20me1, H2BK5me1 and H3K79me1/2/3 in the 5' end of the gene, depletion of the activation marks H2AK5ac, H3K14ac and H3K23ac immediately downstream of the transcription start site (TSS), and characteristic epigenetic asymmetries around the TSS. The chromosome organization factor CTCF may be bound upstream of RNA polymerase in most active CpG island promoters, and an unstable nucleosome at the TSS may be specifically marked by H4K20me3, the first example of such a modification. H3K36 monomethylation is only detected as enriched in the bodies of active genes that have CpG island promoters. Finally, as expression levels increase, peak modification levels of the histone methylations H3K9me1, H3K4me1, H3K4me2 and H3K27me1 shift further away from the TSS into the gene body. CONCLUSIONS: These results suggest that active genes with CpG island promoters have a distinct step-like series of modified nucleosomes after the TSS. The identity, positioning, shape and relative ordering of transcription-associated histone modifications differ between genes with and without CpG island promoters. This supports a model where chromatin organization reflects not only transcription activity but also the type of promoter in which transcription initiates.

<br><br>

</details>

<details>

<summary><strong>Parallel evolution of chordate cis-regulatory code for development.</strong></summary>

<br>

**Autores:**<br> Doglio L, Goode DK, Pelleri MC, Pauls S, Frabetti F, Shimeld SM, Vavouri T, Elgar G

<br><br>

**Enlace a la revista:** <a href="https://doi.org/SO  - PLoS Genet. 2013 Nov;9(11):e1003904. doi: 10.1371/journal.pgen.1003904. Epub 2013  Nov 21." target="_blank">https://doi.org/SO - PLoS Genet. 2013 Nov;9(11):e1003904. doi: 10.1371/journal.pgen.1003904. Epub 2013 Nov 21.</a>

<br><br>

**Resumen:**<br> Urochordates are the closest relatives of vertebrates and at the larval stage, possess a characteristic bilateral chordate body plan. In vertebrates, the genes that orchestrate embryonic patterning are in part regulated by highly conserved non-coding elements (CNEs), yet these elements have not been identified in urochordate genomes. Consequently the evolution of the cis-regulatory code for urochordate development remains largely uncharacterised. Here, we use genome-wide comparisons between C. intestinalis and C. savignyi to identify putative urochordate cis-regulatory sequences. Ciona conserved non-coding elements (ciCNEs) are associated with largely the same key regulatory genes as vertebrate CNEs. Furthermore, some of the tested ciCNEs are able to activate reporter gene expression in both zebrafish and Ciona embryos, in a pattern that at least partially overlaps that of the gene they associate with, despite the absence of sequence identity. We also show that the ability of a ciCNE to up-regulate gene expression in vertebrate embryos can in some cases be localised to short sub-sequences, suggesting that functional cross-talk may be defined by small regions of ancestral regulatory logic, although functional sub-sequences may also be dispersed across the whole element. We conclude that the structure and organisation of cis-regulatory modules is very different between vertebrates and urochordates, reflecting their separate evolutionary histories. However, functional cross-talk still exists because the same repertoire of transcription factors has likely guided their parallel evolution, exploiting similar sets of binding sites but in different combinations.

<br><br>

</details>

<details>

<summary><strong>Genomic and proteomic dissection and characterization of the human sperm chromatin.</strong></summary>

<br>

**Autores:**<br> Castillo J, Amaral A, Azpiazu R, Vavouri T, Estanyol JM, Ballescà JL, Oliva R

<br><br>

**Enlace a la revista:** <a href="https://doi.org/SO  - Mol Hum Reprod. 2014 Nov;20(11):1041-53. doi: 10.1093/molehr/gau079. Epub 2014  Sep 5." target="_blank">https://doi.org/SO - Mol Hum Reprod. 2014 Nov;20(11):1041-53. doi: 10.1093/molehr/gau079. Epub 2014 Sep 5.</a>

<br><br>

**Resumen:**<br> The mammalian spermatozoon has a unique chromatin structure where the majority of DNA is packaged by protamines, while a small fraction (∼8%) remains associated with nucleosomes. However, the chromatin affinity and repertoire of the additional proteins constituting the different sperm chromatin fractions have not yet been explored. To address this we have carried out a genomic and proteomic characterization of human sperm samples subjected to chromatin fractionation using either 0.65 M NaCl extraction followed by EcoRI/BamHI DNA restriction enzyme digestion, or micrococcal nuclease digestion. DNA fractions corresponding to the nucleosome-packaged DNA were sequenced, confirming an appropriate dissection of the sperm chromatin. In addition we detected and sequenced a subnucleosomal particle. Although both fractions were highly enriched at gene promoters, some sequences were found to be exclusively associated with one of those. The results of the proteomic analyses demonstrate that there are two distinct sets of sperm proteins which differ in chromatin affinity. Histone variants, transcription factors, chromatin-associated and modifying proteins involved in regulatory roles were identified as weakly attached to the sperm DNA, whereas proteins with structural roles were identified in the condensed fraction. Many factors, such as the histone lysine demethylase PHF8 identified for the first time in the human sperm cell in this study, were identified exclusively in soluble fraction. Our results provide additional support to the possibility that all of these factors may constitute additional layers of sperm epigenetic information or have structural or regulatory roles transmitted by the sperm cell to the oocyte at fertilization.

<br><br>

</details>

<details>

<summary><strong>Sperm epigenomics: challenges and opportunities.</strong></summary>

<br>

**Autores:**<br> Casas E, Vavouri T

<br><br>

**Enlace a la revista:** <a href="https://doi.org/SO  - Front Genet. 2014 Sep 18;5:330. doi: 10.3389/fgene.2014.00330. eCollection 2014." target="_blank">https://doi.org/SO - Front Genet. 2014 Sep 18;5:330. doi: 10.3389/fgene.2014.00330. eCollection 2014.</a>

<br><br>

**Resumen:**<br> Sperm is a highly differentiated cell type whose function is to deliver a haploid genome to the oocyte. The sperm "epigenomes" were traditionally considered to be insignificant - the sperm is transcriptionally inactive, its genome is packaged in sperm-specific protamine toroids instead of nucleosomes, and its DNA methylation profile is erased immediately post-fertilization. Yet, in recent years there has been an increase in the number of reported cases of apparent epigenetic inheritance through the male germline, suggesting that the sperm epigenome may transmit information between generations. At the same time, technical advances have made the genome-wide profiling of different layers of the sperm epigenome feasible. As a result, a large number of datasets have been recently generated and analyzed with the aim to better understand what non-genetic material is contained within the sperm and whether it has any function post-fertilization. Here, we provide an overview of the current knowledge of the sperm epigenomes as well as the challenges in analysing them and the opportunities in understanding the potential non-genetic carriers of information in sperm.

<br><br>

</details>

<details>

<summary><strong>Paternal diet defines offspring chromatin state and intergenerational obesity.</strong></summary>

<br>

**Autores:**<br> Öst A, Lempradl A, Casas E, Weigert M, Tiko T, Deniz M, Pantano L, Boenisch U, Itskov PM, Stoeckius M, Ruf M, Rajewsky N, Reuter G, Iovino N, Ribeiro C, Alenius M, Heyne S, Vavouri T, Pospisilik JA

<br><br>

**Enlace a la revista:** <a href="https://doi.org/10.1016/j.cell.2014.11.005" target="_blank">https://doi.org/10.1016/j.cell.2014.11.005</a>

<br><br>

**Resumen:**<br> The global rise in obesity has revitalized a search for genetic and epigenetic factors underlying the disease. We present a Drosophila model of paternal-diet-induced intergenerational metabolic reprogramming (IGMR) and identify genes required for its encoding in offspring. Intriguingly, we find that as little as 2 days of dietary intervention in fathers elicits obesity in offspring. Paternal sugar acts as a physiological suppressor of variegation, desilencing chromatin-state-defined domains in both mature sperm and in offspring embryos. We identify requirements for H3K9/K27me3-dependent reprogramming of metabolic genes in two distinct germline and zygotic windows. Critically, we find evidence that a similar system may regulate obesity susceptibility and phenotype variation in mice and humans. The findings provide insight into the mechanisms underlying intergenerational metabolic reprogramming and carry profound implications for our understanding of phenotypic variation and evolution.

<br><br>

</details>

<details>

<summary><strong>The small RNA content of human sperm reveals pseudogene-derived piRNAs complementary to protein-coding genes.</strong></summary>

<br>

**Autores:**<br> Pantano L, Jodar M, Bak M, Ballescà JL, Tommerup N, Oliva R, Vavouri T

<br><br>

**Enlace a la revista:** <a href="https://doi.org/SO  - RNA. 2015 Jun;21(6):1085-95. doi: 10.1261/rna.046482.114. Epub 2015 Apr 22." target="_blank">https://doi.org/SO - RNA. 2015 Jun;21(6):1085-95. doi: 10.1261/rna.046482.114. Epub 2015 Apr 22.</a>

<br><br>

**Resumen:**<br> At the end of mammalian sperm development, sperm cells expel most of their cytoplasm and dispose of the majority of their RNA. Yet, hundreds of RNA molecules remain in mature sperm. The biological significance of the vast majority of these molecules is unclear. To better understand the processes that generate sperm small RNAs and what roles they may have, we sequenced and characterized the small RNA content of sperm samples from two human fertile individuals. We detected 182 microRNAs, some of which are highly abundant. The most abundant microRNA in sperm is miR-1246 with predicted targets among sperm-specific genes. The most abundant class of small noncoding RNAs in sperm are PIWI-interacting RNAs (piRNAs). Surprisingly, we found that human sperm cells contain piRNAs processed from pseudogenes. Clusters of piRNAs from human testes contain pseudogenes transcribed in the antisense strand and processed into small RNAs. Several human protein-coding genes contain antisense predicted targets of pseudogene-derived piRNAs in the male germline and these piRNAs are still found in mature sperm. Our study provides the most extensive data set and annotation of human sperm small RNAs to date and is a resource for further functional studies on the roles of sperm small RNAs. In addition, we propose that some of the pseudogene-derived human piRNAs may regulate expression of their parent gene in the male germline.

<br><br>

</details>

<details>

<summary><strong>Trim28 Haploinsufficiency Triggers Bi-stable Epigenetic Obesity.</strong></summary>

<br>

**Autores:**<br> Dalgaard K, Landgraf K, Heyne S, Lempradl A, Longinotto J, Gossens K, Ruf M, Orthofer M, Strogantsev R, Selvaraj M, Lu TT, Casas E, Teperino R, Surani MA, Zvetkova I, Rimmington D, Tung YC, Lam B, Larder R, Yeo GS, O'Rahilly S, Vavouri T, Whitelaw E, Penninger JM, Jenuwein T, Cheung CL, Ferguson-Smith AC, Coll AP, Körner A, Pospisilik JA

<br><br>

**Enlace a la revista:** <a href="https://doi.org/CIN - Cell. 2016 Jan 28;164(3):341-2. doi: 10.1016/j.cell.2016.01.006. PMID: 26824648" target="_blank">https://doi.org/CIN - Cell. 2016 Jan 28;164(3):341-2. doi: 10.1016/j.cell.2016.01.006. PMID: 26824648</a>

<br><br>

**Resumen:**<br> More than one-half billion people are obese, and despite progress in genetic research, much of the heritability of obesity remains enigmatic. Here, we identify a Trim28-dependent network capable of triggering obesity in a non-Mendelian, "on/off" manner. Trim28(+/D9) mutant mice exhibit a bi-modal body-weight distribution, with isogenic animals randomly emerging as either normal or obese and few intermediates. We find that the obese-"on" state is characterized by reduced expression of an imprinted gene network including Nnat, Peg3, Cdkn1c, and Plagl1 and that independent targeting of these alleles recapitulates the stochastic bi-stable disease phenotype. Adipose tissue transcriptome analyses in children indicate that humans too cluster into distinct sub-populations, stratifying according to Trim28 expression, transcriptome organization, and obesity-associated imprinted gene dysregulation. These data provide evidence of discrete polyphenism in mouse and man and thus carry important implications for complex trait genetics, evolution, and medicine.

<br><br>

</details>

<details>

<summary><strong>The Exonuclease Trex2 Shapes Psoriatic Phenotype.</strong></summary>

<br>

**Autores:**<br> Manils J, Casas E, Viña-Vilaseca A, López-Cano M, Díez-Villanueva A, Gómez D, Marruecos L, Ferran M, Benito C, Perrino FW, Vavouri T, de Anta JM, Ciruela F, Soler C

<br><br>

**Enlace a la revista:** <a href="https://doi.org/CIN - J Invest Dermatol. 2016 Dec;136(12):2337-2339. doi: 10.1016/j.jid.2016.06.628.  PMID: 27884288" target="_blank">https://doi.org/CIN - J Invest Dermatol. 2016 Dec;136(12):2337-2339. doi: 10.1016/j.jid.2016.06.628. PMID: 27884288</a>

<br><br>

**Resumen:**<br> Trex2 is a keratinocyte-specific 3'-deoxyribonuclease that participates in the maintenance of skin homeostasis after DNA damage. Here, we show that this exonuclease is strongly upregulated in human psoriasis, a hyperproliferative and inflammatory skin disease. Similarly, the imiquimod (IMQ)- and Il23-induced mouse psoriasis was associated with a substantial upregulation of Trex2, which was recruited into fragmented chromatin in keratinocytes that were undergoing impaired proliferation, differentiation, and cell death, indicating an important role in DNA processing. Using Trex2 knockout mice, we have found that Trex2 deficiency attenuated IMQ-induced psoriasis-like skin inflammation and enhanced IMQ-induced parakeratosis. Also, Il23-induced ear swelling was diminished in Trex2 knockout mice in comparison with wild-type (wt) mice. Transcriptome analysis identified multiple genes that were deregulated by Trex2 loss after treatment with IMQ. Specifically, immune response genes and pathways normally associated with inflammation were downregulated, whereas those related to skin differentiation and chromatin biology showed increased expression. Interestingly, Trex2 deficiency led to decreased IMQ-induced keratinocyte death via both cell autonomous and noncell autonomous mechanisms. Hence, our data indicate that Trex2 acts as a critical factor in the pathogenesis of psoriasis by promoting keratinocyte apoptosis and enucleation and thereby influencing skin immune responses.

<br><br>

</details>

<details>

<summary><strong>The epigenetic landscape of Alu repeats delineates the structural and functional genomic architecture of colon cancer cells.</strong></summary>

<br>

**Autores:**<br> Jordà M, Díez-Villanueva A, Mallona I, Martín B, Lois S, Barrera V, Esteller M, Vavouri T, Peinado MA

<br><br>

**Enlace a la revista:** <a href="https://doi.org/SO  - Genome Res. 2017 Jan;27(1):118-132. doi: 10.1101/gr.207522.116. Epub 2016 Dec 20." target="_blank">https://doi.org/SO - Genome Res. 2017 Jan;27(1):118-132. doi: 10.1101/gr.207522.116. Epub 2016 Dec 20.</a>

<br><br>

**Resumen:**<br> Cancer cells exhibit multiple epigenetic changes with prominent local DNA hypermethylation and widespread hypomethylation affecting large chromosomal domains. Epigenome studies often disregard the study of repeat elements owing to technical complexity and their undefined role in genome regulation. We have developed NSUMA (Next-generation Sequencing of UnMethylated Alu), a cost-effective approach allowing the unambiguous interrogation of DNA methylation in more than 130,000 individual Alu elements, the most abundant retrotransposon in the human genome. DNA methylation profiles of Alu repeats have been analyzed in colon cancers and normal tissues using NSUMA and whole-genome bisulfite sequencing. Normal cells show a low proportion of unmethylated Alu (1%-4%) that may increase up to 10-fold in cancer cells. In normal cells, unmethylated Alu elements tend to locate in the vicinity of functionally rich regions and display epigenetic features consistent with a direct impact on genome regulation. In cancer cells, Alu repeats are more resistant to hypomethylation than other retroelements. Genome segmentation based on high/low rates of Alu hypomethylation allows the identification of genomic compartments with differential genetic, epigenetic, and transcriptomic features. Alu hypomethylated regions show low transcriptional activity, late DNA replication, and its extent is associated with higher chromosomal instability. Our analysis demonstrates that Alu retroelements contribute to define the epigenetic landscape of normal and cancer cells and provides a unique resource on the epigenetic dynamics of a principal, but largely unexplored, component of the primate genome.

<br><br>

</details>

<details>

<summary><strong>Transgenerational transmission of environmental information in C. elegans.</strong></summary>

<br>

**Autores:**<br> Klosin A, Casas E, Hidalgo-Carcedo C, Vavouri T, Lehner B

<br><br>

**Enlace a la revista:** <a href="https://doi.org/10.1126/science.aah6412" target="_blank">https://doi.org/10.1126/science.aah6412</a>

<br><br>

**Resumen:**<br> The environment experienced by an animal can sometimes influence gene expression for one or a few subsequent generations. Here, we report the observation that a temperature-induced change in expression from a Caenorhabditis elegans heterochromatic gene array can endure for at least 14 generations. Inheritance is primarily in cis with the locus, occurs through both oocytes and sperm, and is associated with altered trimethylation of histone H3 lysine 9 (H3K9me3) before the onset of zygotic transcription. Expression profiling reveals that temperature-induced expression from endogenous repressed repeats can also be inherited for multiple generations. Long-lasting epigenetic memory of environmental change is therefore possible in this animal.

<br><br>

</details>

<details>

<summary><strong>Impaired DNA replication derepresses chromatin and generates a transgenerationally inherited epigenetic memory.</strong></summary>

<br>

**Autores:**<br> Klosin A, Reis K, Hidalgo-Carcedo C, Casas E, Vavouri T, Lehner B

<br><br>

**Enlace a la revista:** <a href="https://doi.org/SO  - Sci Adv. 2017 Aug 16;3(8):e1701143. doi: 10.1126/sciadv.1701143. eCollection 2017  Aug." target="_blank">https://doi.org/SO - Sci Adv. 2017 Aug 16;3(8):e1701143. doi: 10.1126/sciadv.1701143. eCollection 2017 Aug.</a>

<br><br>

**Resumen:**<br> Impaired DNA replication is a hallmark of cancer and a cause of genomic instability. We report that, in addition to causing genetic change, impaired DNA replication during embryonic development can have major epigenetic consequences for a genome. In a genome-wide screen, we identified impaired DNA replication as a cause of increased expression from a repressed transgene in Caenorhabditis elegans. The acquired expression state behaved as an "epiallele," being inherited for multiple generations before fully resetting. Derepression was not restricted to the transgene but was caused by a global reduction in heterochromatin-associated histone modifications due to the impaired retention of modified histones on DNA during replication in the early embryo. Impaired DNA replication during development can therefore globally derepress chromatin, creating new intergenerationally inherited epigenetic expression states.

<br><br>

</details>

<details>

<summary><strong>Double deficiency of Trex2 and DNase1L2 nucleases leads to accumulation of DNA in lingual cornifying keratinocytes without activating inflammatory responses.</strong></summary>

<br>

**Autores:**<br> Manils J, Fischer H, Climent J, Casas E, García-Martínez C, Bas J, Sukseree S, Vavouri T, Ciruela F, de Anta JM, Tschachler E, Eckhart L, Soler C

<br><br>

**Enlace a la revista:** <a href="https://doi.org/10.1038/s41598-017-12308-4" target="_blank">https://doi.org/10.1038/s41598-017-12308-4</a>

<br><br>

**Resumen:**<br> The cornification of keratinocytes on the surface of skin and oral epithelia is associated with the degradation of nuclear DNA. The endonuclease DNase1L2 and the exonuclease Trex2 are expressed specifically in cornifying keratinocytes. Deletion of DNase1L2 causes retention of nuclear DNA in the tongue epithelium but not in the skin. Here we report that lack of Trex2 results in the accumulation of DNA fragments in the cytoplasm of cornifying lingual keratinocytes and co-deletion of DNase1L2 and Trex2 causes massive accumulation of DNA fragments throughout the cornified layers of the tongue epithelium. By contrast, cornification-associated DNA breakdown was not compromised in the epidermis. Aberrant retention of DNA in the tongue epithelium was associated neither with enhanced expression of DNA-driven response genes, such as Ifnb, Irf7 and Cxcl10, nor with inflammation. Of note, the expression of Tlr9, Aim2 and Tmem173, key DNA sensor genes, was markedly lower in keratinocytes and keratinocyte-built tissues than in macrophages and immune tissues, and DNA-driven response genes were not induced by introduction of DNA in keratinocytes. Altogether, our results indicate that DNase1L2 and Trex2 cooperate in the breakdown and degradation of DNA during cornification of lingual keratinocytes and aberrant DNA retention is tolerated in the oral epithelium.

<br><br>

</details>

<details>

<summary><strong>The Polycomb-Dependent Epigenome Controls β Cell Dysfunction, Dedifferentiation, and Diabetes.</strong></summary>

<br>

**Autores:**<br> Lu TT, Heyne S, Dror E, Casas E, Leonhardt L, Boenke T, Yang CH, Sagar, Arrigoni L, Dalgaard K, Teperino R, Enders L, Selvaraj M, Ruf M, Raja SJ, Xie H, Boenisch U, Orkin SH, Lynn FC, Hoffman BG, Grün D, Vavouri T, Lempradl AM, Pospisilik JA

<br><br>

**Enlace a la revista:** <a href="https://doi.org/CIN - Nat Rev Endocrinol. 2018 Aug;14(8):441. doi: 10.1038/s41574-018-0039-8. PMID:  29875378" target="_blank">https://doi.org/CIN - Nat Rev Endocrinol. 2018 Aug;14(8):441. doi: 10.1038/s41574-018-0039-8. PMID: 29875378</a>

<br><br>

**Resumen:**<br> To date, it remains largely unclear to what extent chromatin machinery contributes to the susceptibility and progression of complex diseases. Here, we combine deep epigenome mapping with single-cell transcriptomics to mine for evidence of chromatin dysregulation in type 2 diabetes. We find two chromatin-state signatures that track β cell dysfunction in mice and humans: ectopic activation of bivalent Polycomb-silenced domains and loss of expression at an epigenomically unique class of lineage-defining genes. β cell-specific Polycomb (Eed/PRC2) loss of function in mice triggers diabetes-mimicking transcriptional signatures and highly penetrant, hyperglycemia-independent dedifferentiation, indicating that PRC2 dysregulation contributes to disease. The work provides novel resources for exploring β cell transcriptional regulation and identifies PRC2 as necessary for long-term maintenance of β cell identity. Importantly, the data suggest a two-hit (chromatin and hyperglycemia) model for loss of β cell identity in diabetes.

<br><br>

</details>

<details>

<summary><strong>Assessment of kinship detection using RNA-seq data.</strong></summary>

<br>

**Autores:**<br> Blay N, Casas E, Galván-Femenía I, Graffelman J, de Cid R, Vavouri T

<br><br>

**Enlace a la revista:** <a href="https://doi.org/10.1093/nar/gkz776" target="_blank">https://doi.org/10.1093/nar/gkz776</a>

<br><br>

**Resumen:**<br> Analysis of RNA sequencing (RNA-seq) data from related individuals is widely used in clinical and molecular genetics studies. Prediction of kinship from RNA-seq data would be useful for confirming the expected relationships in family based studies and for highlighting samples from related individuals in case-control or population based studies. Currently, reconstruction of pedigrees is largely based on SNPs or microsatellites, obtained from genotyping arrays, whole genome sequencing and whole exome sequencing. Potential problems with using RNA-seq data for kinship detection are the low proportion of the genome that it covers, the highly skewed coverage of exons of different genes depending on expression level and allele-specific expression. In this study we assess the use of RNA-seq data to detect kinship between individuals, through pairwise identity by descent (IBD) estimates. First, we obtained high quality SNPs after successive filters to minimize the effects due to allelic imbalance as well as errors in sequencing, mapping and genotyping. Then, we used these SNPs to calculate pairwise IBD estimates. By analysing both real and simulated RNA-seq data we show that it is possible to identify up to second degree relationships using RNA-seq data of even low to moderate sequencing depth.

<br><br>

</details>

<details>

<summary><strong>Human sperm displays rapid responses to diet.</strong></summary>

<br>

**Autores:**<br> Nätt D, Kugelberg U, Casas E, Nedstrand E, Zalavary S, Henriksson P, Nijm C, Jäderquist J, Sandborg J, Flinke E, Ramesh R, Örkenby L, Appelkvist F, Lingg T, Guzzi N, Bellodi C, Löf M, Vavouri T, Öst A

<br><br>

**Enlace a la revista:** <a href="https://doi.org/CIN - Nat Rev Endocrinol. 2020 Apr;16(4):200-201. doi: 10.1038/s41574-020-0331-2. PMID:  32066893" target="_blank">https://doi.org/CIN - Nat Rev Endocrinol. 2020 Apr;16(4):200-201. doi: 10.1038/s41574-020-0331-2. PMID: 32066893</a>

<br><br>

**Resumen:**<br> The global rise in obesity and steady decline in sperm quality are two alarming trends that have emerged during recent decades. In parallel, evidence from model organisms shows that paternal diet can affect offspring metabolic health in a process involving sperm tRNA-derived small RNA (tsRNA). Here, we report that human sperm are acutely sensitive to nutrient flux, both in terms of sperm motility and changes in sperm tsRNA. Over the course of a 2-week diet intervention, in which we first introduced a healthy diet followed by a diet rich in sugar, sperm motility increased and stabilized at high levels. Small RNA-seq on repeatedly sampled sperm from the same individuals revealed that tsRNAs were up-regulated by eating a high-sugar diet for just 1 week. Unsupervised clustering identified two independent pathways for the biogenesis of these tsRNAs: one involving a novel class of fragments with specific cleavage in the T-loop of mature nuclear tRNAs and the other exclusively involving mitochondrial tsRNAs. Mitochondrial involvement was further supported by a similar up-regulation of mitochondrial rRNA-derived small RNA (rsRNA). Notably, the changes in sugar-sensitive tsRNA were positively associated with simultaneous changes in sperm motility and negatively associated with obesity in an independent clinical cohort. This rapid response to a dietary intervention on tsRNA in human sperm is attuned with the paternal intergenerational metabolic responses found in model organisms. More importantly, our findings suggest shared diet-sensitive mechanisms between sperm motility and the biogenesis of tsRNA, which provide novel insights about the interplay between nutrition and male reproductive health.

<br><br>

</details>

<details>

<summary><strong>Mechanisms of epigenetic inheritance of variable traits through the germline.</strong></summary>

<br>

**Autores:**<br> Casas E, Vavouri T

<br><br>

**Enlace a la revista:** <a href="https://doi.org/10.1530/REP-19-0340" target="_blank">https://doi.org/10.1530/REP-19-0340</a>

<br><br>

**Resumen:**<br> During the past half century, evidence for inheritance of variable traits has accumulated from experiments in plants and animals and epidemiological studies in humans. Here, we summarize some of the reported cases of epigenetic inheritance and the proposed mechanisms involved in the transmission of non-genetic information between generations in plants, nematodes, flies and mammals. It has long been accepted that information is epigenetically inherited in plants. Although many questions regarding the underlying mechanisms remain to be answered, it is now evident that epigenetic mechanisms are also responsible for the transmission of phenotypes in animals. We highlight similarities and differences between models and species.

<br><br>

</details>

<details>

<summary><strong>Intergenerational Inheritance of Hepatic Steatosis in a Mouse Model of Childhood Obesity: Potential Involvement of Germ-Line microRNAs.</strong></summary>

<br>

**Autores:**<br> Ribas-Aulinas F, Ribo S, Casas E, Mourin-Fernandez M, Ramon-Krauel M, Diaz R, Lerin C, Kalko SG, Vavouri T, Jimenez-Chillaron JC

<br><br>

**Enlace a la revista:** <a href="https://doi.org/10.3390/nu15051241" target="_blank">https://doi.org/10.3390/nu15051241</a>

<br><br>

**Resumen:**<br> Childhood obesity increases the risk of developing metabolic syndrome later in life. Moreover, metabolic dysfunction may be inherited into the following generation through non-genomic mechanisms, with epigenetics as a plausible candidate. The pathways involved in the development of metabolic dysfunction across generations in the context of childhood obesity remain largely unexplored. We have developed a mouse model of early adiposity by reducing litter size at birth (small litter group, SL: 4 pups/dam; control group, C: 8 pups/dam). Mice raised in small litters (SL) developed obesity, insulin resistance and hepatic steatosis with aging. Strikingly, the offspring of SL males (SL-F1) also developed hepatic steatosis. Paternal transmission of an environmentally induced phenotype strongly suggests epigenetic inheritance. We analyzed the hepatic transcriptome in C-F1 and SL-F1 mice to identify pathways involved in the development of hepatic steatosis. We found that the circadian rhythm and lipid metabolic process were the ontologies with highest significance in the liver of SL-F1 mice. We explored whether DNA methylation and small non-coding RNAs might be involved in mediating intergenerational effects. Sperm DNA methylation was largely altered in SL mice. However, these changes did not correlate with the hepatic transcriptome. Next, we analyzed small non-coding RNA content in the testes of mice from the parental generation. Two miRNAs (miR-457 and miR-201) appeared differentially expressed in the testes of SL-F0 mice. They are known to be expressed in mature spermatozoa, but not in oocytes nor early embryos, and they may regulate the transcription of lipogenic genes, but not clock genes, in hepatocytes. Hence, they are strong candidates to mediate the inheritance of adult hepatic steatosis in our murine model. In conclusion, litter size reduction leads to intergenerational effects through non-genomic mechanisms. In our model, DNA methylation does not seem to play a role on the circadian rhythm nor lipid genes. However, at least two paternal miRNAs might influence the expression of a few lipid-related genes in the first-generation offspring, F1.

<br><br>

</details>
